stream-chat-android-client/src/main/java/io/getstream/chat/android/client/uploader/ProgressTracker.kt

```diff
@@ -25,7 +24,7 @@ public class ProgressTracker {
         this.isComplete.value = isComplete
     }
 
-    public fun isComplete(): Flow<Boolean> = isComplete
+    public fun isComplete(): StateFlow<Boolean> = isComplete
```

Comment:
Easier to observe from Jetpack Compose this way

Reply:
This class will be removed after ProgressTrackerFactory refactor. 😉